src/Youtube.php

```diff
@@ -35,10 +35,15 @@ public function youtubeEmbedCode()
             . '" frameborder="0" gesture="media" allow="encrypted-media"'
             . ' allowfullscreen></iframe>';
     }
+    
+    public function youtubeProfile()
+    {
+        return 'https://www.youtube.com/' . $this->youtubeId();
```

Comment:
I'm not sure about using the YouTube Id generator as a profile name. Perhaps you could use one of the other Faker providers such as `Person` (eg. `RachelZulauf`) or `Internet`'s `username` (`wade55`).

Reply:
The goal of this Faker is to get the full value rather than a concatenation of value and another faker. I found also the need for other similar Fakers: playlist, channel... I could send separate PR for each type of URL.

Reply:
I have submitted a separate PR to include channel uri:  #8. Thanks